eps/src/main/java/com/adyen/checkout/eps/EPSConfiguration.kt

```diff
@@ -45,6 +45,17 @@ class EPSConfiguration private constructor(
             clientKey
         )
 
+        /**
+         * Sets whether the logos should be shows next to the issuers name.
+         *
+         * Default is true.
+         *
+         * @param hideIssuerLogos if issuer logos should be hidden or not.
+         */
+        override fun setHideIssuerLogos(hideIssuerLogos: Boolean): Builder {
```

Comment:
Didn't it work without overriding?

Reply:
Not sure how else to do it?

Reply:
I was just curious, because I thought the base class handled it